core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -508,6 +508,7 @@ public static String checkAuthority(String authority) {
   /**
    * Combine a host and port into an authority string.
    */
+  // There is a copy of this method in io.grpc.Grpc
```

Comment:
As commented there, why not just remove this copy and have all the callers call the `io.grpc.Grpc` version?